charter/index.html

```diff
@@ -92,7 +92,8 @@ <h1 id="title">PROPOSED Solid Working Group Charter</h1>
         <p>Any text rendered like <span class="todo">this</span> refers to content that must be updated when the final charter is published at the latest (e.g., adjusting hyperlinks).</p>
       </div>
 
-      <p class="mission">The <strong>mission</strong> of the <a href="">Solid Working Group <i class="todo">(LINK TBD)</i></a> is to standardize the <cite><a href="https://solidproject.org/TR/protocol">Solid Protocol</a></cite> and its use of associated data interoperability and authentication schemes. This effort will culminate in open standards that can be used by developers of servers and applications to continue to build a rich ecosystem that returns control of data back to users.</p>
+      <p class="mission">The <strong>mission</strong> of the <a href="">Solid Working Group <i class="todo">(LINK TBD)</i></a> is to standardize the <cite><a href="https://solidproject.org/TR/protocol">Solid Protocol</a></cite> and its use of associated data interoperability and authentication schemes.
```

Comment:
I know you're breaking the sentence but if you're touching this at all, then as mentioned in  https://github.com/solid/solid-wg-charter/issues/7 , "associated data interoperability" is not meaningful - buzzwording.